developer/src/server/tsconfig.json

```diff
@@ -15,7 +15,8 @@
     "noImplicitAny": true,
     "strictBindCallApply": true,
     "strictFunctionTypes": true,
-    "noUnusedLocals": true
+    "noUnusedLocals": true,
+    "lib": ["es6"]
```

Comment:
> The "dom" type definitions provided a different type spec for that in `@types/node`, which was the source of the conflict. As `developer/server` compiles down to Node modules (commonjs), it's meant for command-line and not DOM, so dropping those type definitions should be safe.

Sounds good to me. Given we are now node18, can we target a newer version than ES6?

Reply:
Oh, almost certainly so.  I just went with something I knew was safe there.

Reply:
After a little hunting, I came upon this link:  https://kangax.github.io/compat-table/es2016plus/

It appears to indicate we could say ES2022.